gen/com/intellij/plugins/haxe/lang/psi/HaxeAbstractClassDeclaration.java

```diff
@@ -31,8 +31,8 @@
   @Nullable
   HaxeComponentName getComponentName();
 
-  @Nullable
-  HaxeFunctionType getFunctionType();
+  @NotNull
```

Comment:
I believe that these sorts of changes are fundamentally incorrect, as evidenced by the addition of getFirstItem() that you had to add to all of your refactoring below.  Why should an element that is supposed to exist in the singular be parsed into a list?  Personally, I think this is a bug in Grammar-kit, which is one reason why I've resisted moving forward to newer versions of it.  However, it has been like this for a couple of years, and it's very annoying to deal with, so I won't stop this change.